admin/package.json

```diff
@@ -141,7 +141,8 @@
   "standard": {
     "ignore": [
       "assets/javascripts/jquery-3.5.1.js",
-      "assets/vendor-js/*"
+      "assets/vendor-js/*",
+      "**/*.ts"
```

Comment:
I thought `standard` could lint ts files.  Why ignore them?

Reply:
not completely - https://github.com/standard/standard#can-i-use-a-javascript-language-variant-like-flow-or-typescript

if you open the admin app as the root in your editor, you get syntax errors.
we should assess this carefully with eslint and figure out the best path.
i will create a ticket for that